etna/models/holt_winters.py

```diff
@@ -609,18 +616,47 @@ def __init__(
             )
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid doesn't tune ``seasonal_periods`` parameter. It expected to be set by the user.
 
-class HoltModel(HoltWintersModel):
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        if self.seasonal is None:
+            return {
+                "trend": CategoricalDistribution(["add", "mul", None]),
+                "damped_trend": CategoricalDistribution([False, True]),
+                "use_boxcox": CategoricalDistribution([False, True]),
```

Comment:
In case of `use_boxcox=True` and negative target model will throw an error.

Reply:
Yes, but current signature doesn't allow us to work around this problem. So, I think we will just allow it to fail.